TASK/TaskGoToCoordAnyMeansExtraParamsWithCruiseSpeed.md

```diff
@@ -5,25 +5,105 @@ ns: TASK
+
+  // Drives normally except for the fact that it ignores lights
+  DRIVINGMODE_STOPFORCARS_IGNORELIGHTS = 786475, // DF_StopForCars|DF_SteerAroundStationaryCars|DF_StopForPeds|DF_SteerAroundObjects|DF_UseShortCutLinks|DF_ChangeLanesAroundObstructions
+
+  // Try to swerve around everything, but stop for lights if necessary
+  DRIVINGMODE_AVOIDCARS_OBEYLIGHTS = 786597, // DF_SwerveAroundAllCars|DF_StopAtLights|DF_SteerAroundObjects|DF_UseShortCutLinks|DF_ChangeLanesAroundObstructions|DF_StopForCars
+
+  // Swerve around cars, be careful around peds, and stop for lights
+  DRIVINGMODE_AVOIDCARS_STOPFORPEDS_OBEYLIGHTS = 786599 // DF_SwerveAroundAllCars|DF_StopAtLights|DF_StopForPeds|DF_SteerAroundObjects|DF_UseShortCutLinks|DF_ChangeLanesAroundObstructions|DF_StopForCars
+};
+
+enum TaskGoToCoordsAnyMeansFlags {
+	TGCAM_DEFAULT									= 0,
+	// Ignore the health of the vehicle (default behaviour is to not use any vehicle with less than 600 health)
+	TGCAM_IGNORE_VEHICLE_HEALTH						= 1,
+	// Considers all nearby vehicles for suitability (default behaviour is to consider only the vehicle closest to the ped)
+	TGCAM_CONSIDER_ALL_NEARBY_VEHICLES				= 2,
+	// Performs the same tests as is done in IS_VEHICLE_DRIVEABLE (default behaviour is to just check the vehicle's health)
+	TGCAM_PROPER_IS_DRIVEABLE_CHECK					= 4,
+	// Instructs the ped to remain in the vehicle at the end, so that multiple tasks can be chained together (see url:bugstar:1778387)
+	TGCAM_REMAIN_IN_VEHICLE_AT_DESTINATION			= 8,
+	// Ped will never abandon the vehicle it is in (see url:bugstar:2196034)
+	TGCAM_NEVER_ABANDON_VEHICLE						= 16,
+	// Ped will never abandon the vehicle it is in if vehicle is moving (see url:bugstar:2225456)
+	TGCAM_NEVER_ABANDON_VEHICLE_IF_MOVING			= 32,
+	// Peds will use the targeting system for threats and register any threats they can attack (rather than just using the closest targetable ped)
+	TGCAM_USE_AI_TARGETING_FOR_THREATS				= 64
+};
 ```
 
 ## Parameters
-* **ped**: 
-* **x**: 
-* **y**: 
-* **z**: 
-* **speed**: 
-* **p5**: 
-* **p6**: 
-* **walkingStyle**: 
-* **p8**: 
-* **p9**: 
-* **p10**: 
-* **p11**: 
-* **p12**: 
+* **ped**: The `Ped` Handle.
+* **x**: The goto target coordinate.
+* **y**: The goto target coordinate.
+* **z**: The goto target coordinate.
+* **fMoveBlendRatio**: 0.0 = still, 1.0 = walk, 2.0 = run, 3.0 = sprint.
+* **vehicle**: If defined, the pedestrian will only move if said vehicle exists. If you don't want any sort of association, just set it to `0`.
+* **bUseLongRangeVehiclePathing**: Setting to `true` tells the vehicle to use longrange vehicle pathing.
+* **drivingFlags**: See `eDrivingMode` enum.
+* **fMaxRangeToShootTargets**: Determines the maximum distance at which the `Ped` will engage in combat with threatening targets.
+* **fExtraVehToTargetDistToPreferVeh**:
+* **fDriveStraightLineDistance**: 
+* **iExtraFlags**: See `TaskGoToCoordAnyMeansFlags` enum.
```

Comment:
Ok, where do I find the enum you are talking about? 🙂

Here you go:

```c
enum eTaskGoToCoordAnyMeansFlags {
  TGCAM_DEFAULT = 0,
  // Ignore the health of the vehicle (default behaviour is to not use any vehicle with less than 600 health)
  TGCAM_IGNORE_VEHICLE_HEALTH = 1,
  // Considers all nearby vehicles for suitability (default behaviour is to consider only the vehicle closest to the ped)
  TGCAM_CONSIDER_ALL_NEARBY_VEHICLES = 2,
  // Performs the same tests as is done in IS_VEHICLE_DRIVEABLE (default behaviour is to just check the vehicle's health)
  TGCAM_PROPER_IS_DRIVEABLE_CHECK = 4,
  // Instructs the ped to remain in the vehicle at the end, so that multiple tasks can be chained together.
  TGCAM_REMAIN_IN_VEHICLE_AT_DESTINATION = 8,
  // Ped will never abandon the vehicle it is in.
  TGCAM_NEVER_ABANDON_VEHICLE = 16,
  // Ped will never abandon the vehicle it is in if vehicle is moving.
  TGCAM_NEVER_ABANDON_VEHICLE_IF_MOVING = 32,
  // Peds will use the targeting system for threats and register any threats they can attack (rather than just using the closest targetable ped)
  TGCAM_USE_AI_TARGETING_FOR_THREATS = 64
};
```

Reply:
Isn't the enum already in this file ?